ui/components/src/ui-components/properties/renderers/PropertyRenderer.tsx

```diff
@@ -9,14 +9,15 @@
 import * as React from "react";
 import { PropertyRecord, PropertyValueFormat } from "@bentley/ui-abstract";
 import { Orientation, RatioChangeResult } from "@bentley/ui-core";
+import { HighlightedRecordProps } from "../../../ui-components";
```

Comment:
Should not import from barrel file of the same repo

Reply:
Changed